problem.py

```diff
@@ -36,10 +36,13 @@ def regex_find_href(tag, exp):
         subs = []
         for tag in soup.find_all('div', {'class': 'submission-row'}):
             if 'AC' in tag.find('div', {'class': 'sub-result'})['class']:  # Fake AC is not counted
-                user_id = regex_find_href(tag, r'/user/(\w+)').group(1)
-                sub_id = regex_find_href(tag, r'/submission/(\d+)').group(1)
-                lang = tag.find('span', {'class': 'language'}).text
-                subs.append(Submission(sub_id, user_id, lang))
+                try:
+                    user_id = regex_find_href(tag, r'/user/(\w+)').group(1)
+                    sub_id = regex_find_href(tag, r'/submission/(\d+)').group(1)
+                    lang = tag.find('span', {'class': 'language'}).text
+                    subs.append(Submission(sub_id, user_id, lang))
+                except:
+                    pass
```

Comment:
![](https://i.kym-cdn.com/photos/images/facebook/001/416/688/925.jpg_large)

Reply:
I couldn't think of a better way to do this, since otherwise we'd need to check if the user had ac'd each problem individually and there's no specific API endpoint for that.

Reply:
Rather, I've now edited it to log the error message to assist in debugging if something goes wrong.